.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,7 @@
   "image": "mcr.microsoft.com/devcontainers/typescript-node:dev-22",
   "onCreateCommand": "sed -i 's/\r$//' ../../.devcontainer/post-create.sh && sed -i  's/\r$//' ../../.devcontainer/on-create.sh && bash ../../.devcontainer/on-create.sh",
   "postCreateCommand": "bash ../../.devcontainer/post-create.sh",
-  "postAttachCommand": "nohup bash -c 'kubectl port-forward svc/postgres 5432:5432 &'",
+  "postStartCommand": "bash ../../.devcontainer/curbside-pickup/post-start.sh",
```

Comment:
Why is this root level `devcontainer.json` file using the `post-start.sh` script that is in the curbside folder?